src/GiftRanking.tsx

```diff
@@ -84,6 +82,10 @@ const GiftRanking = () => {
     setSearchParams({ gender: selectedTab, category: subTab }, { replace: true });
   };
   
+  const [visible,setvisible]=useState(6);
```

Comment:
```suggestion
  const [visible,setVisible]=useState(6);
```
camelCase 규칙을 잘 지켜주시는게 좋습니다 😁